youtube_dl/postprocessor/ffmpeg.py

```diff
@@ -451,7 +451,7 @@ def add(meta_list, info_list=None):
         # 4. http://atomicparsley.sourceforge.net/mpeg-4files.html
 
         add('title', ('track', 'title'))
-        add('date', 'upload_date')
+        add('date', ('release_year', 'upload_date'))
```

Comment:
This will set a year rather than the yyyymmdd value if both items are available. Shouldn't they be reversed?

Reply:
I think not. This field called "Year" in most media players.